# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdelibs/sasl2.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- kdelibs-4.1.0/cmake/modules/FindSasl2.cmake.vanilla	2008-09-03 10:41:14.000000000 +0200
+++ kdelibs-4.1.0/cmake/modules/FindSasl2.cmake	2008-09-03 10:41:39.000000000 +0200
@@ -16,7 +16,7 @@
   set(SASL2_FIND_QUIETLY TRUE)
 endif (SASL2_INCLUDE_DIR)
 
-FIND_PATH(SASL2_INCLUDE_DIR sasl/sasl.h
+FIND_PATH(SASL2_INCLUDE_DIR sasl2/sasl.h
 )
 
 FIND_LIBRARY(SASL2_LIBRARIES NAMES sasl2
